components/scream/data/scream_input.yaml

```diff
@@ -61,31 +77,32 @@ HOMME:
     partmethod        : 4
     cubed_sphere_map  : 0
     topology          : cube
-    test_case         : jw_baroclinic
     u_perturb         : 1
     rotate_grid       : 0
     ne                : 4
     statefreq         : 9999
     mesh_file         : /dev/null
     tstep             : 300
-    nu                : 7.0e+15
-    nu_div            : 1.0e+15
-    nu_p              : 7.0e+15
-    nu_q              : 7.0e+15
-    nu_s              : -1
-    nu_top            : 2.5e+5
+    nu                :  .000000034
+    nu_div            : -1 
+    nu_p              : .000000034
+    nu_q              : .000000034
+    nu_s              : .000000034
+    nu_top            : 
```

Comment:
Did you mean to leave `nu_top` empty here?

Reply:
ooops, good catch.  I'll fix that